src/sensors/scripts/car_image_raw.py

```diff
@@ -21,11 +21,13 @@ def airpub():
 
     while not rospy.is_shutdown():
          # get camera images from the car
-        responses = client.simGetImages([
-            airsim.ImageRequest("1", airsim.ImageType.Scene, False, False)])  #scene vision image in uncompressed RGB array
+        #responses = client.simGetImages([
```

Comment:
Is there a particular reason this is commented out? Seems to be the same as the line 27.

Reply:
oh woops that was just me messing around trying to debug something, turned out my airsim version was off will fix